### PR TITLE
Transmission with VPN, WebProxy - v8.5.6

### PR DESCRIPTION
**Pull request by ChaosZero112**
### OpenVPN and Transmission with WebUI

This container contains OpenVPN and Transmission with a configuration where Transmission is running only when OpenVPN has an active tunnel. It bundles configuration files for many popular VPN providers to make the setup easier.

You need to specify your provider and credentials with environment variables, as well as mounting volumes where the data should be stored. An example run command to get you going is provided below.

It also bundles an installation of Tinyproxy to also be able to proxy web traffic over your VPN, as well as scripts for opening a port for Transmission if you are using PIA or Perfect Privacy providers.

https://github.com/haugene/docker-transmission-openvpn

### CODE REVIEW

File: apps/transmission.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    - name: 'RSS feed configuration - Folder'
+      block:
+        - name: 'Creating RSS folder'
+          shell: "mkdir /opt/appdata/{{pgrole}}/rss"
+      when: not rsscheck.stat.exists
+
+    - name: 'RSS feed configuration - Config'
+      block:
+        - name: 'Creating configuration file'
+          shell: "touch /opt/appdata/{{pgrole}}/rss/transmission-rss.conf && chown -R 1000:1000 /opt/appdata/{{pgrole}}/rss/ && echo see https://git.io/fhAm2 to configure > /opt/appdata/{{pgrole}}/rss/transmission-rss.conf"
```
Comment (robertbaker) [inline apps/transmission.yml]: This could download the same file instead

Comment (ChaosZero112) [reply apps/transmission.yml]: Ah looks like that was left over when I was testing. But don't need it, transmission handles the gz unpack.

Comment (Admin9705) [reply apps/transmission.yml]: whatya think @robertbaker ?

Comment (robertbaker) [reply apps/transmission.yml]: If it's not needed then remove the block

Comment (ChaosZero112) [reply apps/transmission.yml]: Commented out blocks have been removed.

**Pull request merged**